ugwumadu116/Auto-Mart-Andela
# #166116390 build out user can get cars of a specific body type

#### What does this PR do?
- Creates a route to get all cars with specific body type
#### Description of Task to be completed?
- Create a route that uses GET /car/ eg `GET /api/v1/car?body_type=car`
- Create a car controller to handle route request
- Create a car validator to know the user that is making the request
#### How should this be manually tested?
- Sign in with firstName, lastName, email, address, and password
   - exmaple `{
	"firstName": "joel",
	"lastName": "ugwumadu2",
	"email": "ugwumadu116@gmail.com",
	"password": "test"
	"address": "ipaja road"

}`
- Login in with the your email and password 
  - example `{
	"email": "ugwumadu116@gmail.com",
	"password": "test"
}`
- Use your login token and set it in Headers and set the key to be authorization while the value will be your token
- Then make a GET request to `/api/v1/car?body_type=car `
- Your messages will appear in your response
#### What are the relevant pivotal tracker stories?
#166116390
